proto/panacea/oracle/v2alpha2/oracle.proto

```diff
@@ -75,8 +75,9 @@ message TallyResult {
   string no = 2 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
   // Votes yes, but the value is different with the consensus value
   string invalid_yes = 3 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  string total = 4 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
```

Comment:
There are cases where no votes are cast, so total is added to calculate the quorum.